frontend/src/@common/components/Header/Header.tsx

```diff
@@ -31,15 +31,6 @@ const Header = ({ handleViewModeChange }: HeaderProps) => {
           onClick={() => navigate('/')}
           css={buttonStyle}
         ></HomeButton>
```

Comment:
```suggestion
        />
```

자식 요소가 없어서 self closing이면 충분할 것 같습니다!